release notes/v0.49.0.md

```diff
@@ -17,9 +17,13 @@ k6 `v0.49.0` is here 🎉! This release includes:
 
 _optional intro here_
 
-### `<big_feature_1>` `#pr`
+### Add tracing to browser module [browser#1100](https://github.com/grafana/xk6-browser/pull/1100)
 
-_what, why, and what this means for the user_
+The browser module now generates traces that provide a representation of its inner workings, such as API methods executed (for example `browser.newPage` and `page.goto`), page navigations, and [Web Vitals](https://grafana.com/docs/k6/latest/using-k6-browser/metrics/#googles-core-web-vitals) measurements.
+
```

Comment:
I was thinking on adding an image here of how a browser trace can is visualized in Tempo, but I see it is not a common practice in release notes to add images. Should we add a folder under `release notes/` for resources that are referenced in the release notes? Or is the text description good enough?

Reply:
Agreed, a visual representation of this would be useful. Maybe best for the release email that goes out after the release instead of here?

Reply:
What are your thoughts on this @mstoykov ?